python/kserve/kserve/protocol/rest/openai/types/openapi.py

```diff
@@ -0,0 +1,3461 @@
+# Copyright 2024 The KServe Authors.
```

Comment:
Could we ignore this file from linter and license generate check?

Reply:
To ignore it from autoformatting you can add a rule here: https://github.com/kserve/kserve/blob/master/python/pyproject.toml#L4

And for linting add a rule here: https://github.com/kserve/kserve/blob/master/.flake8

Reply:
Thanks! 
Any config to ignore the Verify Generated Code / verify-codegen check here?